plugins/modules/rabbitmq_user_limits.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2013, Chatham Financial <oss@chathamfinancial.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: rabbitmq_user_limits
+short_description: Manage RabbitMQ user limits
+description:
+  - Manage the state of user limits in RabbitMQ
+author: Aitor Pazos (@aitorpazos)
+options:
+  user:
+    description:
+      - Name of user to manage limits for
+    type: str
+    required: true
+    aliases: [username, name]
+  max_connections:
+      description:
+          - Max number of concurrent client connections.
+          - Negative value means "no limit".
+          - Ignored when the I(state) is C(absent).
+      type: int
+      default: -1
+  max_channels:
+      description:
+          - Max number of channels.
+          - Negative value means "no limit".
+          - Ignored when the I(state) is C(absent).
+      type: int
+      default: -1
+  node:
+      description:
+          - Name of the RabbitMQ Erlang node to manage.
+      type: str
+  state:
+      description:
+          - Specify whether the limits are to be set or cleared.
+          - If set to C(absent), the limits of both I(max_connections) and I(max_channels) will be cleared.
+      type: str
+      default: present
+      choices: [present, absent]
+'''
+
+EXAMPLES = '''
+# Limit both of the max number of connections and channels on the user 'guest'.
+- community.rabbitmq.rabbitmq_user_limits:
```

Comment:
```suggestion
- name: Limit both of the max number of connections and channels on the user 'guest'
  community.rabbitmq.rabbitmq_user_limits:
```
This is the doc requirement, please do same for the other examples

Reply:
This should be fixed